content-security-policy/generic/quote-characters.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+  <script>
+    var tests = [
+      { "csp": "img-src 'none';", "expected": false, "name": "precondition: single quotes are accepted" },
+      // See https://github.com/w3c/webappsec-csp/issues/434 for background on what is tested.
+      { "csp": 'img-src "none";', "expected": true, "name": "double quotes should not be accepted" },
```

Comment:
In Chromium, this treats the source `"none"` as an unknown source, leaving us with `img-src`, which is an empty list, which is the same as `'none'`.